manifest-v2.json

```
@@ -204,6 +204,41 @@
       "useCaseName": "Multi-step workflow"
     }
   ],
+  "go (provided.al2)": [
+    {
+      "directory": "go1.x/cookiecutter-aws-sam-hello-golang",
```

Just an update to the path (for all the new al2 golang templates)

```suggestion
      "directory": "provided.al2/go/cookiecutter-aws-sam-hello-golang",
```

---

Thanks for catching! I thought I'd done this edit 🤦‍♂️